src/main/java/com/github/dockerjava/api/command/InspectContainerResponse.java

```diff
@@ -225,17 +225,32 @@ public String toString() {
     @JsonIgnoreProperties(ignoreUnknown = true)
     public class ContainerState {
 
+        @JsonProperty("Status")
+        private String status;
+        
         @JsonProperty("Running")
         private Boolean running;
 
         @JsonProperty("Paused")
         private Boolean paused;
 
+        @JsonProperty("Restarting")
+        private Boolean restarting;
+        
+        @JsonProperty("OOMKilled")
+        private Boolean oomKilled;
+        
+        @JsonProperty("Dead")
+        private Boolean dead;
```

Comment:
Please see:
https://docs.docker.com/v1.8/reference/api/docker_remote_api_v1.17/
"State": { *"Error": ""**, * "ExitCode": 9, "FinishedAt":
"2015-01-06T15:47:32.080254511Z", *"OOMKilled": false**, * "Paused": false,
"Pid": 0, *"Restarting": false**, * "Running": false, "StartedAt":
"2015-01-06T15:47:32.072697474Z" },

Also, while the docker document does not have it, I see some additional
properties in the latest 1.21 (Docker 1.9 API). I can remove them if you
strictly want to go by the documentation but usually docker documentation
is always behind..

    "State": {

*        "Status": "exited",*

        "Running": false,

        "Paused": false,

        "Restarting": false,

        "OOMKilled": false,

*        "Dead": false,*

        "Pid": 0,

        "ExitCode": 0,

        "Error": "",

        "StartedAt": "2015-11-20T19:02:48.972024978Z",

        "FinishedAt": "2015-11-20T19:02:48.998958231Z"

    }


On Fri, Nov 20, 2015 at 10:54 AM, Kanstantsin Shautsou <
notifications@github.com> wrote:

> In
> src/main/java/com/github/dockerjava/api/command/InspectContainerResponse.java
> <https://github.com/docker-java/docker-java/pull/372#discussion_r45503748>
> :
>
> >          @JsonProperty("Running")
> >          private Boolean running;
> >
> >          @JsonProperty("Paused")
> >          private Boolean paused;
> >
> > +        @JsonProperty("Restarting")
> > +        private Boolean restarting;
> > +
> > +        @JsonProperty("OOMKilled")
> > +        private Boolean oomKilled;
> > +
> > +        @JsonProperty("Dead")
> > +        private Boolean dead;
>
> Didn't found in 1.17..1.21 please point on API documentation
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker-java/docker-java/pull/372/files#r45503748>.
>



-- 
Ritesh
Founder @ Nirmata <http://nirmata.com/>